packages/@n8n/plugin-sdk/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@n8n/plugin-sdk",
+  "version": "0.0.0",
+  "type": "module",
```

Comment:
Some packages we have specify `type: module`, but most don't. When do we use which?

Reply:
We should always use `type: module` where possible. It enables modern js code in node files (using `import`/`export` instead of `require`/`module.exports`). Some of our older packages are not currently migrated to the new syntax.